math/wxMaxima/Portfile

```diff
@@ -1,112 +1,78 @@
-# openssl is a dependency of curl in cmake
-license_noconflict  openssl
+license_noconflict  openssl # openssl is a dependency of curl in cmake, so build only
 platforms           darwin
 categories          math aqua
 description         Graphical user interface for Maxima based on wxWidgets
-homepage            http://andrejv.github.io/wxmaxima/
-
-subport wxMaxima-devel {
-    conflicts       wxMaxima
-    long_description ${description}: \
-                    Provides the development version, which is typically updated every day.
-
-    # git describe --tags : Version-16.12.x-1834-gb32fed0
-    github.setup    andrejv wxmaxima b32fed0d535c05d589d1f573d2c0f4f839180d18
-    checksums       rmd160  86fcf85ddae42e84884a708d55acff393152ed23 \
-                    sha256  e34256294c51a37d80947b92d3cd979456bf0c8ae53a21ce6cbab63d99143bf2 \
-                    size    12739940
-    version         18.02-dev-20180616
-    revision        1
-
-    patchfiles      patch-devel-src_Dirstructure.cpp.diff
-    post-patch {
-        reinplace -W ${worksrcpath} "s|@@PREFIX@@|${prefix}|g" \
-                                    src/Dirstructure.cpp
-    }
-
-    livecheck.type  none
-}
+homepage            https://github.com/wxMaxima-developers/wxmaxima
+long_description    ${description}: version ${version}
 
-if {${subport} eq ${name}} {
-    conflicts       wxMaxima-devel
-    long_description ${description}: \
-                    Provides the release version, which is typically updated every four months.
-
-    # get the source tarball from github.
-    github.setup    andrejv wxmaxima 18.02.0 Version-
-    checksums       rmd160  d1017d56bfd76e9c51dcc3230be89ebca1724560 \
-                    sha256  77f0414be54d8ea15acedfc3cedf72a57cc2b3ba5d4835a53f3009e050c05483 \
-                    size    10131293
-    revision        1
-
-    patchfiles      patch-src_Dirstructure.cpp.diff
-    post-patch {
-        reinplace -W ${worksrcpath} "s|@@PREFIX@@|${prefix}|g" \
-                                    src/Dirstructure.cpp
-    }
-    
-    livecheck.regex Version-(\[a-z0-9.\]+)${extract.suffix}
-}
+github.setup        wxMaxima-developers wxmaxima ${version} Version-
+checksums           rmd160  0c10602f6f1d777ff16fe85500f57222eb07fc27 \
+                    sha256  055ff34eb98539716a18f7cc11861b5a790fde46aaafabb68f1e690cb649d9df \
+                    size    15314478
 
-universal_variant   no
+livecheck.regex     Version-(\[a-z0-9.\]+)${extract.suffix}
 
-variant wx32 description {Use wxWidgets 3.2 - experimental!} {}
+universal_variant   no
 
-if {[variant_exists wx32] && [variant_isset wx32]} {
-    # we want to use the wxWidgets-3.2
-    wxWidgets.use   wxWidgets-3.2
-} else {
-    wxWidgets.use   wxWidgets-3.0
-}
+# Note: as of Match 2020 (1b00466)the build with wxWidgets-3.2 has serious issues with scrolling
+wxWidgets.use       wxWidgets-3.0
 
-depends_lib         port:${wxWidgets.port}
+depends_lib         port:${wxWidgets.port} port:libomp-devel
```

Comment:
- Change `depends_lib` to `depends_lib-append` to not overwrite any portgroup dependencies. 
- We don't like to depend on `-devel` ports. Does the `libomp` port really not work here? 
- Add the `libomp` dependency on another line.

Reply:
Thanks for the detailed review - I appreciate it!

A question: is libomp-devel an unreleased build / master branch build of libomp or is it a package variant which includes besides the library also C the headers (as e.g. in many linux distros)? I am not 100% sure but I think I tried port:libomp and I had issues with missing header files, so I assumed it was a version without headers. Indeed I would not like to depend on something like a master branch build.

Btw.: even the libomp-devel version is many years old.

Reply:
The former. `-devel` ports are simply intended for the latest development version. But they often don't get up to date with even that. 

In Macports, all ports are supposed to have the development headers. It would be considered a bug if they don't. You might have to open a Trac ticket against the libomp port. The good news is that it has a maintainer so it will hopefully get fixed.

Reply:
Indeed libomp works fine. I guess when I tested it I was missing one of the 10 extra cmake settings one needs when one uses libomp, tried libompdevel and then forgot to revert this. I changed it now.